### ISSUE TITLE
Move to wordpress.org

### ISSUE THREAD

**Issue by SDavisMedia** [easydigitaldownloads/edd-widgets-pack#14]
I still do not feel like the move to .org is the right choice. But if we're going to do it, couple things to do here:

- [ ] If the other issues are important, let's get them fixed up ASAP: https://github.com/easydigitaldownloads/edd-widgets-pack/issues
- [ ] Widget audit... I don't feel all of the widgets need to be in the extension. See below...

1. I think Top Sellers, Featured Download, and Random Download should move to EDD core.
2. I think Related Downloads should be scrapped. We have paid extensions for this.
3. I think the rest can stay in the pack, which makes it super weak... which is why I suggest EDD Library

If you all (mainly @pippinsplugins) do not agree with number 1 above, then yes, off to .org the entire extension will go.

**Comment by pippinsplugins** [comment]
@SDavisMedia what is your ideal move? And, specifically, what about moving to .org is wrong in your opinion?

**Comment by SDavisMedia** [comment]
It's not the move to .org that bothers me. It's making the decision to move to .org based on the current collection of widgets that I don't like. Here are the 8 widgets and what I think the fate of some of them should be:

* Top Sellers - move to core
* Featured Download - move to core
* Random Download - move to core
(keep in mind I'm saying this alongside the effort to improve core a bit as we slowly change business models)

* Related Downloads - discontinue, conflicts with Recommended Products & Cross-sell Upsell
* Most Commented - discontinue, let's not promote download comments ... promote Reviews instead

That leaves just...

* Most Recent
* Downloads Archive
* Downloads Calendar

Just like we did with all the extensions, I think the widgets deserve that audit as well. Those top 5 suggestions there are for improvement of EDD core and the company itself, which I think is more important than keeping the widgets pack together.

If we all agree on that, leaving just a few stranded widgets left behind, I see no better place for them than the EDD Library.

So basically, the reason I don't want to see it go to .org as a whole is because I want to see specific widgets go to other places more, if that makes sense.

**Comment by spencerfinnell** [comment]
From the outside looking in I would agree with @SDavisMedia 100% -- the widgets that don't conflict with add-ons should be moved to core and the other (from what I could imagine) not so popular ones dropped (or put in the snippet library).

**Comment by rubengc** [comment]
I also agree with @sdavismedia moving some widgets to core adds an small extra and other widgets should be at their respectives ones

Keep in mind #12 for the process
